tests/k8s/multi-node/cluster/cluster-manager.bash

```diff
@@ -319,76 +294,30 @@ function deploy_cilium(){
 
     rm "${cilium_dir}/cilium-lb-ds.yaml" \
        "${cilium_dir}/cilium-ds.yaml" \
-       "${cilium_dir}/cilium-ds-1.yaml" \
-       "${cilium_dir}/cilium-ds-2.yaml" \
         2>/dev/null
 
     if [[ -n "${lb}" ]]; then
-        node_selector='"kubernetes.io/hostname": "cilium-k8s-master"'
         # In loadbalancer mode we set the snoop and LB interface to
         # enp0s8, the interface with IP 192.168.36.11.
         iface='enp0s8'
 
         sed -e "s+\$disable_ipv4+${disable_ipv4}+g;\
-                s+\$node_selector+${node_selector}+g;\
                 s+\$iface+${iface}+g" \
             "${cilium_dir}/cilium-lb-ds.yaml.sed" > "${cilium_dir}/cilium-lb-ds.yaml"
 
-        node_selector='"kubernetes.io/hostname": "cilium-k8s-node-2"'
-        node_address='#'
-        ipv4_range='#'
-
-        sed -e "s+\$disable_ipv4+${disable_ipv4}+g;\
-                s+\$node_selector+${node_selector}+g;\
-                s+\$node_address+${node_address}+g;\
-                s+\$ipv4_range+${ipv4_range}+g;\
-                s+\$server_number+1+g;\
-                s+\$iface+${iface}+g" \
-            "${cilium_dir}/cilium-ds.yaml.sed" > "${cilium_dir}/cilium-ds.yaml"
-
         kubectl create -f "${cilium_dir}"
 
-        wait_for_daemon_set_ready kube-system cilium-server-1 1
+        wait_for_daemon_set_ready kube-system cilium-lb 2
```

Comment:
2 cilium-lb? The cilium-lb is a daemon-set to test k8s ingress with a specific node selector label. There should be a single ingress controller running in the k8s cluster otherwise we will end up with conflicting IPs when trying to update the frontend IP in the kubernetes ingress resource.